src/edu/csus/ecs/pc2/clics/API202306/EventFeedJSON.java

```diff
@@ -1056,13 +1069,80 @@ public void setFilter(Filter filter) {
         this.filter = filter;
     }
 
+    /**
+     * Check if we are using collections for notifications
+     *
+     * @return true if collections are enabled, false otherwise
+     */
     public boolean isUseCollections() {
         return useCollections;
     }
 
+    /**
+     * Set if we want to try to use collections when possible for notifications
+     *
+     * @param useCollections - true to try to use collections
+     */
     public void setUseCollections(boolean useCollections) {
         this.useCollections = useCollections;
     }
+
+    /**
+     * Tests if notification collections should be used for a specific notification type.
+     * This first tests is notification collections are being used in general, then if so,
+     * checks if the specific type is not disabled.
+     *
+     * @param notificationType
+     * @return
+     */
+    public boolean isUseNotificationCollection(EventFeedType notificationType) {
+        return(isUseCollections() && !disableCollections.contains(notificationType.toString()));
+    }
+
+    /**
+     * Adds the comma-separated list of notification types (EventFeedType's) to the HashSet of
+     * disabled notification types.  Any notification type in this list will not use collections
+     * on the event feed.  The special type ALL_NOTIFICATION_TYPES will disable collections for all
+     * notification and will clear the HashSet of disabled types and simply set the useCollections flag to false.
+     *
+     * @param list Comma-separated list of notification types (EventFeedType's) to not use
+     *      collections on.
+     */
+    public void disableNotificationCollections(String [] list) {
+        if(list != null) {
+            for(String notificationType : list) {
+                if(notificationType.equals(ALL_NOTIFICATION_TYPES)) {
+                    setUseCollections(false);
+                    disableCollections.clear();
+                    break;
+                }
+                disableCollections.add(notificationType);
+            }
+        }
+    }
+
+    /**
+     * Remove the comma-separated list of notification types (EventFeedType's) from the Hashset
+     * of disabled notification types.  If collections are enabled, then the types included in the
+     * supplied list will be allowed to generate collections.  The special type ALL_NOTIFICATION_TYPES will disable collections for all
+     * notification and will simply set the useCollections flag to true and clear the HashSet of
+     * disabled types.
+     *
+     * @param list Comma-separated list of notification types (EventFeedType's) to allow collections on.
```

Comment:
As previously mentioned, referring to a "comma-separated list" here is confusing, because that's not what the method is receiving.

Reply:
Fixed in next push.